.env

```diff
@@ -0,0 +1 @@
+LOCALSTACK_DOCKER_NAME=localstack
```

Comment:
I would not recommend adding a default `.env` file; if going for the option to set a container name, specify the default in the compose file instead; something like:

```
${LOCALSTACK_DOCKER_NAME-localstack}
```

`.env` files are not supported when using `docker stack deploy`. In addition, users may want to set their own `.env` file either to customize, or for other purposes

Reply:
Gotcha ty for the feedback! I'll go ahead and remove this